.github/workflows/run_test.yaml

```diff
@@ -27,35 +27,31 @@ jobs:
 
```

Comment:
@Nicole00 we could add 3.6 3.7 3.8 back in separate PR after this being merged. :D

Reply:
AFAIK the `setup-python` action can no longer install 3.6 **easily**. And some of our dev dependencies don't work on 3.6. https://github.com/vesoft-inc/nebula-python/blob/2eeaef4f5daee904d250945f69c00cc4c44df043/pyproject.toml#L32-L42

Reply:
Ah, got it, let's put CI with 3.9+ as is, and keep the requirement broader to defer the decision of changing minimal py.

We could make CI best effort in a slightly broader in other PR.

@Nicole00 This seems to be a good enough timing for at least dropping 3.6 :D